stream_alert/shared/rule_table.py

```diff
@@ -73,15 +73,16 @@ def __str__(self, verbose=False):
 
         return '\n'.join(output)
 
-    def _add_new_rules(self):
+    def _add_new_rules(self, skip_staging=False):
         """Add any new local rules (renamed rules included) to the remote database"""
         # If the table is empty, no rules have been added yet
         # Add them all as unstaged to avoid demoting rules from production status
-        init = (len(self.remote_rule_names) == 0)
+        # Also, allow the user to bypass staging with the skip_staging flag
+        skip_staging = skip_staging or (len(self.remote_rule_names) == 0)
         with self._table.batch_writer() as batch:
             for rule_name in self.local_not_remote:
-                LOGGER.debug('Adding rule \'%s\' (init=%s)', rule_name, init)
-                batch.put_item(self._dynamo_record(rule_name, init))
+                LOGGER.debug('Adding rule \'%s\' (init=%s)', rule_name, skip_staging)
```

Comment:
Forgot to change `init=%s` to `skip_staging=%s`?

Reply:
good catch! updating now